### ISSUE TITLE
[BUG]: It's skipping almost every relevant job

### ISSUE THREAD

**Issue by Pointfit** [AIHawk-app/Auto_Jobs_Applier#756]
### Describe the bug

I have a clear list of jobs to apply to with NO blacklisted jobs

### Steps to reproduce

positions:
  - Data
  - Analytics
  - Analytics Engineer
  - Software engineer
  - Software Developer
  - Data Analyst
  - Data engineer
  - Data scientist
  - SQL
  - Software Engineer I
  - Developer

### Expected behavior

Applying to all these jobs

### Actual behavior

Skipping them, sometimes even applying to irrelevent jobs

### Branch

None

### LLM Used

OpenAI

### Model used

gpt-4o-mini

**Comment by 49Simon** [comment]
This should not happen, there is a feature where it sends the job description and your resume to the LLM you're using and it scores your suitability for the job. It only applies for scores >= 7. 

- Can you share screenshot of the logs and see what the score and reasoning is for the jobs it is not applying?
- You can change the [condition](https://github.com/AIHawk-app/Auto_Jobs_Applier/blob/0fd49db4d6bab364f1f5dbe18b79e6f5ba5051a4/src/ai_hawk/llm/llm_manager.py#L631) to a value less than 7 and apply to jobs with lower suitability score.

**Comment by sirraht** [comment]
I'm seeing the same issue.  I'm on branch release/v11.01.2024.

AIHawk is skipping jobs that I'm looking to apply to.

In config.yaml, I specify:

positions:
  - Head of Professional Services
  - VP of Professional Services
  - VP Professional Services
  - Director of Professional Services
  - Director Professional Services
  
  I'm attaching my "skipped.json" and "success.json".  HawkAI is applying successfully to jobs with lowest condition value, skipping my targeted job. 
[skipped.json](https://github.com/user-attachments/files/17653626/skipped.json)
[success.json](https://github.com/user-attachments/files/17653627/success.json)

**Comment by Pointfit** [comment]
@49Simon you are right, after reading the logs more carefully the LLM's reasoning makes sense as to why it is not applying.

I guess my feature request would be a more organized log of LLM's reasoning to apply/not apply to jobs.

Also for the change of code:
I would change both the 7's into a 6 for example, in that function.

**Comment by 49Simon** [comment]
In the next release, there will be a `JOB_SUITABILITY_SCORE` variable where users can change it to any value they want. This will give flexibility to apply to more jobs. 

I don't think there's a bug related to this issue at this moment.

**Comment by surapuramakhil** [comment]
@49Simon can you ping me on discord
